src/Slices/ServiceInventory/UI/Tabs/Tabs.tsx

```diff
@@ -24,14 +29,27 @@ export enum TabKey {
 }
 
 interface Props {
-  activeTab: TabKey;
-  setActiveTab: (tabKey: TabKey) => void;
+  activeTab: TabKey | string;
```

Comment:
I think it would be helpful to add info why we escape enum into general type, at first I was confused but then I checked into DocumentationTab and realised that it's probably due to the fact that we can't foresee every possibility there. But to realise that I had to look into another file

Reply:
added in the docstrings